pyproject.toml

```diff
@@ -37,7 +38,7 @@ where = ["src"]
 version = {attr = "aind_metadata_upgrader.__version__"}
 
 [tool.black]
-line-length = 79
+line-length = 120
 target_version = ['py36']
```

Comment:
We may need to update the template also. This needs to be py38 and the requires python line above needs to be >=3.8